src/components/Cabinet/Cabinet.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import PersonOutlineIcon from '@material-ui/icons/PersonOutline';
+
+import { useStyles } from './Cabinet.styles';
+import NotLoggedCabinet from '../NotLoggedCabinet';
+import LoggedCabinet from '../LoggedCabinet';
+
+const Cabinet = () => {
+  const isLogged = false;
+  const props = {
```

Comment:
Why do you need this variable? Can you just write `useStyles({ isLogged })` and get rid of `props`

Reply:
Changed